web/app/plugins/rkg-plugin/includes/RKGeronimoExcursion.php

```diff
@@ -125,16 +125,44 @@ public function rkgShowExcursionData()
             $tableName          = $wpdb->prefix."rkg_course_meta";
             $firstJoin          = $wpdb->prefix."rkg_course_template";
             $secondJoin         = $wpdb->prefix."posts";
-            $context['courses'] = $wpdb->get_results("SELECT rcm.id, rcm.category AS cat, "
+
+            $whereClause = "WHERE rcm.endtime > '".date("Y-m-d")."' ";
+            if (!current_user_can('edit_others_courses')) {
```

Comment:
Took me a while to get it. Would add comment here that admin can reserve for any course (if that was the intent and I am reading correctly)